tools/nni_cmd/updater.py

```diff
@@ -14,7 +14,7 @@
 def validate_digit(value, start, end):
     '''validate if a digit is valid'''
     if not str(value).isdigit() or int(value) < start or int(value) > end:
-        raise ValueError('%s must be a digit from %s to %s' % (value, start, end))
+        raise ValueError('%s must be a digit from %s to %s' % ('value', start, end))
```

Comment:
it is strange that you add quote on `value`

Reply:
I think there used to be a typo here. If the value is 1, it's strange to raise a error like ValueError('1 must be a digit from 1 to 1000' % (value, start, end)). Instead, it should be ValueError('value must be a digit from 1 to 1000' % (value, start, end))

Reply:
then it should be `raise ValueError('value must be a digit from %s to %s' % (start, end))`. i still suggest the following:
`raise ValueError('value (%s) must be a digit from %s to %s' % ('value', start, end))`

Reply:
Goog idea. Will fix.